tests/Application/config/jwt/private.pem

```diff
@@ -0,0 +1,54 @@
+-----BEGIN ENCRYPTED PRIVATE KEY-----
```

Comment:
Should be commited?

Reply:
In my opinion yes, because a private key exists and is used only in a test environment. We can change the name to `private-test.pem` to a self-descriptive convention like in e.g. SyliusInvoicingPlugin https://github.com/Sylius/InvoicingPlugin/blob/master/tests/Application/config/jwt/private-test.pem what do you think about it?

Reply:
it's fine, keep it